CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## v2.0.0.pre
+
+- Major change: RSpec plugin to use HTTP Upload API instead of websocket connection to send test data #174 #175 - @niceking
+- `identifier` field removed from trace #176 - @amybiyuliu
```

Comment:
Missing #160

Only warn on EOF errors and also catch SSLErrors

```suggestion
- `identifier` field removed from trace #176 - @amybiyuliu
- Only warn on EOF errors and also catch SSLErrors #160 - @gchan
```

Reply:
The #160 PR went in like a month ago and there's been 5 releases with that change since then? You're welcome to open another PR to put #160 in the changelog for v1.3.0 but I'm not going to include it here because it isn't a change between v1.5.0 and v2.0.0

Reply:
@niceking I don't think #160 ever made it into a release? It was merged after the release of v1.5.0!